devices/pine64-pinephone-braveheart/kernel/0001-dts-pinephone-Setup-default-on-and-panic-LEDs.patch

```diff
@@ -1,36 +1,39 @@
-From 32d81a88ad076538d4d1e4ac88dda01fc2165b30 Mon Sep 17 00:00:00 2001
-From: Samuel Dionne-Riel <samuel@dionne-riel.com>
-Date: Sat, 28 Mar 2020 01:33:16 -0400
+From a9c97de0e0465427653c302a03b6f1ba7bc78bdb Mon Sep 17 00:00:00 2001
+From: S Dao <si.dao@outlook.com>
+Date: Tue, 14 Jul 2020 17:34:13 +0700
 Subject: [PATCH] dts: pinephone: Setup default on and panic LEDs
 
  * The green LED defaults to on.
  * The red LED is the panic indicator.
 
 The green LED defaults to on for an expected transition through red,
 yellow and green during the different boot stages.
+
+Signed-off-by: S Dao <si.dao@outlook.com>
```

Comment:
(Personal peeve) I would have liked to see `Co-authored-by` since you changed the author of the patch. Though I do see the patch has relatively non-trivial changes too, so this is why changing the author isn't an issue in itself.